straxen/corrections_services.py

```diff
@@ -77,66 +79,76 @@ def get_corrections_config(self, run_id, config_model=None):
         """
 
         if not isinstance(config_model, (tuple, list)) or len(config_model) != 2:
-            raise ValueError(f'config_model {config_model} must be a tuple')
-        model_type, global_version = config_model
+            raise ValueError(f'config_model {config_model} must be a tuple of length 2')
+        model_type, version = config_model
```

Comment:
This will not work as the arguments are different right? The config passed to is_cmt_option is a tuple of length 3 (includes the True/False for nT) and config passed here is length 2.